CHANGELOG.md

```diff
@@ -2,10 +2,13 @@
 
 ## [v2.3.2 - Mercury Vulture Fixed Formatting](https://github.com/nf-core/tools/releases/tag/2.3.2) - [2022-03-24]
 
-Very minor patch release to fix the full size AWS tests and re-run the template sync, which partially failed due to GitHub pull-requests being down at the time of release.
+This patch release to removes the Graphviz dependency from default pipeline template.
+
```

Comment:
Actually `2.3.2` has already been released, there should have been a  `## dev` header for you to put that in

Reply:
Just wing it

Reply:
Just one merge conflict had to be resolve. Done.